unit-tests/src/test/scala/scala/scalanative/unsafe/CArrayOpsTest.scala

```diff
@@ -14,7 +14,7 @@ class CArrayOpsTest {
     val ptr   = alloc.asInstanceOf[Ptr[Int]]
 
     assertTrue(ptr == carr.at(0))
-    (1 to 100).foreach { i => assertTrue((ptr + i) == carr.at(i)) }
+    (1 to 31).foreach { i => assertTrue((ptr + i) == carr.at(i)) }
```

Comment:
Shouldn't 32 elements be from 0 to 31?

Reply:
@ekrich `0` is tested at line 16 :)

Reply:
Ok thanks. Did not look at the whole file.